content/articles/files-and-exeptions-in-python/index.md

```diff
@@ -0,0 +1,265 @@
+ 
+
```

Comment:
Could the introduction be more elaborative? 

Talk in-detail about Files and exceptions in Python, what their uses are, and talk about what the reader is going to learn, what are the key takeaways and outcomes.

Have a look at other Section articles.